utils/inspect-response.js

```diff
@@ -1,5 +1,5 @@
 
 const util = require('util');
 module.exports = (response) => {
-    return util.inspect(response, { showHidden: false, depth: null, colors: true });
+    return util.inspect(response, { showHidden: true, depth: null, colors: true, maxArrayLength: null });
```

Comment:
why is `showHidden` needed?

Reply:
I found it useful since it printed out array length.